### PR TITLE
feat: add builder-projects-panel scenes search bar

### PR DESCRIPTION
**Pull request by pbosio**
Splitting feature Builder Projects Panel in several PR to avoid a huge PR:

* add Builder Projects Panel search bar

### CODE REVIEW

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/SearchBarView/SortDropdownView.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    public void Show()
+    {
+        if (!gameObject.activeSelf)
+        {
+            gameObject.SetActive(true);
+        }
+
+        if (!(EventSystem.current is null))
```
Comment (sandrade-dcl) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/SearchBarView/SortDropdownView.cs]: Maybe this line could be clearer in this way?

```suggestion
        if (EventSystem.current is not null)
```

**Review by sandrade-dcl — APPROVED** [review]
A very clear code, congratulations :)

**Review by D4rWiNSS — APPROVED** [review]
Good job! LGTM!

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/SearchBarView/SortDropdownView.cs (hunk 1 above)
Comment (pbosio) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/SearchBarView/SortDropdownView.cs]: That's not supported by c# versions < 9 :(
I changed it for `!= null`

**Pull request merged**